pkg/conduit/client.go

```diff
@@ -1,4 +1,4 @@
-package public
+package conduit
```

Comment:
I think you are right about the coupling. I was originally worried about not having "too smart" client libraries, but ultimately this lib is smart because it abstracts gRPC over HTTP and extension points. I moved it back in latest commits.